plugins/slicer/MONAILabel/MONAILabel.py

```diff
@@ -25,6 +25,8 @@
 import SimpleITK as sitk
 import sitkUtils
 import slicer
+from slicer.i18n import tr as _
```

Comment:
is it a good idea to use _?
in python _ is normally used while declaring unused variable names (returns, tuples etc..)

@lassoan ?

Reply:
We were surprised, too, that `_()` is the standard name for translation function in Python, but [that's how it is](https://docs.python.org/3/library/gettext.html), it is a [well-known potential issue](https://peps.python.org/pep-0640/). In practice it does not interfere with returned dummy values because if you overwrite `_` in a function then it only has effect there, so you only need to pay attention on not using `_` for different purposes within a function.